scout/server/blueprints/institutes/templates/overview/institute.html

```diff
@@ -0,0 +1,201 @@
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('overview.institutes') }}">
+      institutes
+    </a>
+  </li>
+  <li class="nav-item active">
+    <p class="navbar-text text-nowrap">{{ institute.display_name }}</p>
+  </li>
+{% endblock %}
+
+{% block content_main %}
+<form  action="{{ url_for('overview.institute', institute_id=institute.internal_id) }}" method="POST"
+  accept-charset="utf-8" id="institute_form">
+{{ form.csrf_token() }}
+<div class="container">
+  <div class="display:block">
+    <!-- institute settings panel -->
+    <div class="card mt-3">
+      <div style="padding: 0;" class="card-body mt-3 ml-3 mr-3" >
+        <h5 class="header">Institute Settings</h5>
+          <div class="form-row mt-3 d-flex justify-content-between">
+            <div class="form-group col-md-12">
+              {{ form.display_name.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Name of the institite which will be displayed on Scout pages") }}
+              {{ form.display_name(class='form-control', value=institute.display_name) }}
+              {% for error in form.display_name.errors %}
+                <span style="color: red;">{{ error }}</span><br>
+              {% endfor %}
+            </div>
+          </div>
+          <div class="form-row mt-3 mb-3">
+            <div class="col-md-2">
+              {{form.sanger_emails.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Email addresses to send variant verification email to")}}
+            </div>
+            <div class="col-md-10">
+              <select class="select2" id="sanger_tags" name="sanger_emails" multiple="true" style="width:100%;">
+                {% for email in institute.sanger_recipients %}
+                  <option value="{{ email }}" selected>{{ email }}</option>
+                {% endfor %}
+              </select>
+              {% for error in form.sanger_emails.errors %}
+                <span style="color: red;">{{ error }}</span><br>
+              {% endfor %}
+            </div>
+          </div>
+          <div class="form-row mt-3 d-flex justify-content-between">
+            <div class="form-group col-md-5">
+              {{ form.coverage_cutoff.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Read depth considered reliable for displaying variants") }}
+              {{ form.coverage_cutoff(class='form-control', value=institute.coverage_cutoff) }}
+              {% for error in form.coverage_cutoff.errors %}
+                <span style="color: red;">{{ error }}</span><br>
+              {% endfor %}
+            </div>
+            <div class="form-group col-md-5">
+              {{ form.frequency_cutoff.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Lower frequency threshold to display variants") }}
```

Comment:
NOTE : used as the institutet default in clinical filter.

Reply:
Should I write also 'NOTE'?

Reply:
If it fits the tooltip, why not - some kind of emphasis since it does change something rather crucial. Oh, and "institute" - I've had to unlearn that because KI.. :)

Reply:
Ah ok, I'll add it!